src/main.c

```diff
@@ -1,6 +1,9 @@
 #include <pebble.h>
 
 #define WORKER_TICKS 0
+#define INSET 8
+#define TICKS_LAYER_ALIGNMENT PBL_IF_RECT_ELSE(GTextAlignmentLeft, GTextAlignmentCenter)
```

Comment:
If these defines are only used in one location, probably better to put them inline.